admin_manual/groupware/calendar.rst

```diff
@@ -148,15 +148,15 @@ Rate limits
 Nextcloud rate limits the creation of calendars and subscriptions if too many items are created within a short time frame. The default is 10 calendars or subscriptions per hour. This can be customized as follows::
 
   # Set limit to 15 items per 30 minutes
-  php occ config:app:set calendar rateLimitCalendarCreation --type=integer --value=15
-  php occ config:app:set calendar rateLimitPeriodCalendarCreation --type=integer --value=1800
+  php occ config:app:set calendar rateLimitCalendarCreation --value=15
+  php occ config:app:set calendar rateLimitPeriodCalendarCreation --value=1800
```

Comment:
```suggestion
  php occ config:app:set dav rateLimitCalendarCreation --value=15
  php occ config:app:set dav rateLimitPeriodCalendarCreation --value=1800
```

Reply:
Yep. I'll cherry-pick https://github.com/nextcloud/documentation/pull/11636 onto this branch